plugins/harness-srm/src/components/Icons.tsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+
+export const HeartIcon = () => (
```

Comment:
Following icons not available in Material UI?

Reply:
No these are from core ui harness.